lib/commands/powershell.js

```diff
@@ -0,0 +1,115 @@
+import _ from 'lodash';
+import { fs, tempDir } from 'appium-support';
+import { errors } from 'appium-base-driver';
+import { exec } from 'teen_process';
+import log from '../logger';
+import path from 'path';
+import B from 'bluebird';
+
+const EXECUTION_POLICY = {
+  REMOTE_SIGNED: 'RemoteSigned',
+  UNDEFINED: 'Undefined',
+  RESTRICTED: 'Restricted',
+};
+const POWER_SHELL = 'powershell.exe';
+const POWER_SHELL_SCRIPT_PATTERN = /^powerShell$/;
+const POWER_SHELL_FEATURE = 'power_shell';
+
+const commands = {};
+
+commands.execute = async function execute (script, args) {
+  if (POWER_SHELL_SCRIPT_PATTERN.test(script)) {
+    this.ensureFeatureEnabled(POWER_SHELL_FEATURE);
```

Comment:
👍

Reply:
Add this in readme like https://github.com/appium/appium-xcuitest-driver#opt-in-features-with-security-risk ?

Reply:
Kazu, you know how much I like to write documentation %)

Reply:
haha :P
(I can assist later)